jmbitcoin/jmbitcoin/bip21.py

```diff
@@ -0,0 +1,41 @@
+# https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki
+# bitcoin:<address>[?amount=<amount>][?label=<label>][?message=<message>]
+# We don't check validity of Bitcoin address here, as all the tools using
+# this are expected to do address validation independently anyway.
+
+from jmbitcoin import amount_to_sat
+from urllib.parse import parse_qs, urlparse
```

Comment:
Is urllib available in Py3 by default or is this  dependency to add to setup.py?

Reply:
Yes, it looks to me so, it's in cpython codebase. https://github.com/python/cpython/tree/3.8/Lib/urllib